lambdas/data-migration1/tests/test-collections.js

```diff
@@ -17,15 +17,22 @@ const {
   migrateCollections,
 } = require('../dist/lambda');
 
-const knex = Knex({
+const knexAdminConfig = {
   client: 'pg',
   connection: {
     host: 'localhost',
     user: 'postgres',
     password: 'password',
     database: 'postgres',
   },
-});
+};
+
+// eslint-disable-next-line node/no-unpublished-require
```

Comment:
I'm not entirely hot on this - we might consider moving the actual migration logic out of the lambda.

Reply:
Yeah, I'm inclined to agree about moving the actual migration logic into the db package. I don't really love referencing bundle handler code directly. @marc?

Reply:
Thinking about this, I think it warrants a separate PR.      I'll update this to fix the table truncation concern for now and issue a second PR to address the unit test concern.

Reply:
Since the "migration logic" is only `knex.migrate.latest();`, I'm not particularly inclined to move that to a separate package. Could this test just call `knex.migrate.latest();`, rather than referencing the handler?

Reply:
It also needs to load the migrations based on `env.migrationDir` and I'm loathe to start replicating all the logic from the handler throughout unit tests

Reply:
I don't think I understand what we're talking about, then. Depending on where the test file is located, we are going to have to reference _something_ that knows where the migrations are located, and that something is going to be different depending on where the test file is located.

In this case (and using the connection / config changes in my PR), it would look something like:

```
  t.context.knex = await getKnexClient({
    ...localStackConnectionEnv,
    migrationDir: path.join(__dirname, '..', '..', 'db-migration', 'dist', 'migrations'),
  });

  await t.congext.knex.migrate.latest();
```

Is the logic that we're concerned about replicating the location of the migrations dir?

Reply:
That seems workable as a pattern so long as 

1) the migration lambda's 'latest' includes literally nothing else.    
2) unit tests don't need to utilize any of the other commands (they shouldn't)

Reply:
OK, I defer to you here

Reply:
383289200 looks pretty good to me